lib/tests/facts/schema.cc

```diff
@@ -560,6 +561,13 @@ void validate_attributes(YAML::Node const& node)
         REQUIRE_FALSE(caveats.empty());
     }
 
+    auto blockgroup_attribute = node["blockgroup"];
```

Comment:
We could create new collections with the blockgroup specified, and confirm that the fact doesn't resolve. It doesn't seem unreasonable to test mapping between facts and resolvers, though.